book/src/tutorial/rust-std.md

```diff
@@ -0,0 +1,73 @@
+# Combining Rust `std` with `uefi`
+
+## TL;DR
+
+In Mid-2024, we recommend to stick to our normal guide. Use this document as
+guide and outlook for the future of UEFI and Rust.
+
+## About
+
+In the `uefi` crate, we provide tooling to create a `no_std` + `no_main`
+binary, i.e., an UEFI image, in a convenient way. However, there is also the
+option to create a "standard" binary with Rust. The result is the same, but the
+build process and the overall programming is much more similar to regular
```

Comment:
nit: the build process is the same either way right? `cargo build --target <uefi-target>`